test/tests/hylang-sh/container.hy

```diff
@@ -4,8 +4,6 @@
   (+ 2 2)
   (print))
 
-(import subprocess sys)
-(subprocess.check_call [sys.executable "-m" "pip" "install" "-q" "sh==1.14.2"])
```

Comment:
This moves the pip install to the main shell file to remove the extra output causing the standard output to not match

https://doi-janky.infosiftr.net/job/update.sh/job/pypy/10376/execution/node/98/log/

```
failed; unexpected output:
--- /mnt/docker/jenkins/doi-janky/workspace/update.sh/pypy/oi/test/tests/python-hy/expected-std-out.txt	2023-02-24 15:07:07.700935269 -0800
+++ -	2023-10-16 15:08:31.525198206 -0700
@@ -1,2 +1,6 @@
+/opt/pypy/lib/pypy3.10/site-packages/pip/_vendor/certifi/cacert.pem None
+/opt/pypy/lib/pypy3.10/site-packages/pip/_vendor/certifi/cacert.pem None
+/opt/pypy/lib/pypy3.10/site-packages/pip/_vendor/certifi/cacert.pem None
+/opt/pypy/lib/pypy3.10/site-packages/pip/_vendor/certifi/cacert.pem None
 4
 42
```

Reply:
Ah, the random `/opt/pypy/lib/pypy3.10/site-packages/pip/_vendor/certifi/cacert.pem None` debug lines that are breaking the test have an issue filed and fix applied: https://foss.heptapod.net/pypy/pypy/-/issues/4010, but not certain when that'll be in a pypy release.

I guess it makes sense to hide the stdout for the `pip` installs, since this test is more about the `container.hy` running and producing the correct output. 👍